Cookbook-ios/AppDelegate.swift

```diff
@@ -20,6 +21,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         let rootViewController = UINavigationController(rootViewController: viewController)
         window?.rootViewController = rootViewController
         window?.makeKeyAndVisible()
+        GMSPlacesClient.provideAPIKey("AIzaSyAMaZ8NbGqgl0afSID-vdCn_lZIzyDwDTA")
```

Comment:
Just for future reuse for all developers, add a comment indicating which Key we need to use here